lib/Installation/Popups/OKPopupController.pm

```diff
@@ -40,4 +40,9 @@ sub accept {
     $self->get_ok_popup()->press_ok();
 }
 
+sub is_ok_popup_visible {
+    my ($self) = @_;
+    return ($self->{OKPopup}->is_shown() ? 1 : 0);
```

Comment:
why to re-map to 1 or 0, `is_shown` always returns 1 or 0  using `exist` from `lib/YuiRestClient/Widget/Base.pm`, right?

Reply:
I had tried and it would always go inside the unless condition, but I will try again.

Reply:
seems like it works now for tests that the popup doesn't appear as well :+1: